introduction.md

```diff
@@ -0,0 +1 @@
+Hi Deadly_Viprers, My name is Mike Wyatt :-)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.